modules/ui/fields/combo.js

```diff
@@ -498,7 +544,16 @@ export function uiFieldCombo(field, context) {
                 .attr('maxlength', maxLength);
 
         } else {
-            utilGetSetValue(input, displayValue(tags[field.key]));
+            var isMixed = Array.isArray(tags[field.key]);
+
+            var mixedValues = isMixed && tags[field.key].map(function(val) {
+                return displayValue(val);
+            }).filter(Boolean);
+
+            utilGetSetValue(input, !isMixed ? displayValue(tags[field.key]) : '')
+                .attr('title', isMixed ? mixedValues.join('; ') : undefined)
```

Comment:
Ooo good idea! I didn't know you could use newlines in tooltips 😄

Reply:
Yeah, it’s kind of a hidden feature. In HTML you actually have to put in character entity references for a carriage return and/or line feed, because any whitespace just ends up getting treated as a space. But in JavaScript a standard newline should be sufficient.